lib/index.js

```diff
@@ -246,6 +279,82 @@ export class Baker extends EventEmitter {
     }
   }
 
+  /**
+   * Prepares a function to process crosswalk image paths.
+   * @param {string} crosswalkPath - The base path for crosswalk images.
+   * @returns {Function} A function that maps over data entries to update image paths.
+   */
+  prepareCrosswalk(/** @type {string} */ crosswalkPath) {
+    /**
+     * Processes data entries to set appropriate paths for images.
+     * @param {Object.<string, string>[]} data - Array of objects, each containing image references.
+     * @returns {Object.<string, string>[]} The data with updated image paths.
+     */
+    return function setCrosswalkPath(
+      /** @type {Object.<string, string>[]} */ data
+    ) {
+      /** @type {Object.<string, string>[]} */ data
+      const isAbsolutePath =
+        isProductionEnv &&
+        process.env.BAKER_BASE_PATH &&
+        process.env.BAKER_PATH_PREFIX;
+      const extWhitelist = ['img', 'avif', 'webp'];
+
+      return data.map((d) => {
+        extWhitelist.forEach((ext) => {
+          if (d[ext]) {
+            const relativeStaticPath = this.getStaticPath(
+              join(crosswalkPath, d[ext])
+            );
+            const absoluteUrlSrc = new URL(
+              relativeStaticPath,
+              normalizeUrlPaths(this.domain)
+            ).href;
+
+            return {
+              ...d,
+              [ext]: isAbsolutePath ? absoluteUrlSrc : relativeStaticPath,
+            };
+          }
+        });
+
+        return { ...d };
+      });
+    };
+  }
+
+  /**
+   * Sends a webhook to notify of build completion
+   * @returns {Promise<void>}
+   */
+  async sendWebhook() {
+    if (!process.env.BUILD_NOTIFICATION_WEBHOOK_URL) return;
+
+    const files = await glob(join(this.embeds, '*.html'));
+    if (files.length === 0) return;
+
+    const pathObject = path.parse(files[0]);
+    const pathArray = pathObject.dir.split(path.sep);
+    const projectSlug = pathArray[pathArray.length - 2];
```

Comment:
Can we get the needed information from the current rollup/build context? Since this is a class instance member we have access to the `this` object.

Reply:
those values are based of: `await glob(join(this.embeds, '*.html'));` Where it specifically points to whats inside the embeds folder. Without this specificity, we won't have access to the file names or `embed` slug